### ISSUE TITLE
External Wireless Adapters?

### ISSUE THREAD

**Issue by gstone787** [bluecross-dev/kernel_module_wlan-fw-api#1]
#question
Forgive my ignorance on the subject. I'm looking for a kernel or a kernel mod that will allow me to use an external wifi adapter with monitor mode via USB OTG cable for using my TP Link TL-WN722N with Nethunter and Andrax. Will this file do that for me? I've tried following tutorials to compile the kernel myself but haven't had any luck.

**Comment by artem** [comment]
Hi!
This repository only contains definitions of the WMI protocol used for integrated Wi-Fi chip.
To support your USB dongle, you have to compile and load drivers designed for it (fast googling suggests the driver is called rtl8188eu, it can be obtained from the TP-Link site or other resources)

**Comment by gstone787** [comment]
Thanks for the response. I've tried compiling a kernel with the driver's added and enabled. But I always either get an error while compiling or an error while flashing. Not sure what I'm doing wrong. I'm not a developer but I'm pretty good at following instructions and copy pasting but I've tried several different methods and I've still not been successful. I'm sure I'm just missing something because I'm not knowledgeable in this field but I wanted to give it a shot anyways. I was just hoping that maybe that was what this was. I appreciate your insight and if you ever happen to build a kernel with the external adapters enabled as well as HID patch for ducky scripts just let me know. I'd be glad to donate. Basically I'm needing a standard Nethunter kernel for Pixel 3.
